workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -98,13 +118,40 @@ public static Request CreateRequest(Entity entity, EntityId? entityId = null, ui
                 };
             }
 
+            /// <summary>
+            ///     An object that represents the response of a CreateEntity command from the SpatialOS runtime.
+            /// </summary>
             public struct ReceivedResponse
             {
+                /// <summary>
+                ///     The status code of the command response. Can be used to check whether the command succeeded and
+                ///     denotes the type of failure if the command failure.
+                /// </summary>
                 public StatusCode StatusCode { get; }
+
+                /// <summary>
+                ///     The failure message of the command. Will only be non-null if the command failed.
+                /// </summary>
                 public string Message { get; }
+
+                /// <summary>
+                ///     The Entity ID of the created entity. Will only be non-null if the command succeeded.
```

Comment:
We are inconsistent on EntityId vs Entity ID in comments. I think this one is correct.

Reply:
Yes - I meant to use this notation, but may have neglected to do so in places